src/lib.rs

```diff
@@ -31,6 +31,8 @@
 //! - [char]
 //! - [String]
 //! - [axum::body::Bytes]
+//! - [chrono::DateTime<chrono::FixedOffset>]
```

Comment:
I would add the version here just in case.

Reply:
you mean the version of chrono?